cyhy/mailer/CyhyNotificationMessage.py

```diff
@@ -32,14 +32,18 @@ class CyhyNotificationMessage(ReportMessage):
     TextBody = """Greetings {{acronym}},
 
 Cyber Hygiene scans of your host(s) conducted in the past day have detected one or both of the following:
-* New critical and/or high vulnerabilities
+* New critical, high, and/or known exploited vulnerabilities
 * New potentially risky services
 
 {{#is_federal}}As part of BOD 19-02, critical findings need to be remediated within 15 days and high findings remediated within 30 days.{{/is_federal}}{{^is_federal}}CISA recommends remediating critical findings within 15 days and high findings within 30 days.{{/is_federal}}
 
-CISA also recommends reviewing hosts with potentially risky open services (e.g. RDP, Telnet, etc.) to ensure that each service is intended to be available to the public and, where applicable, the service is up-to-date on the latest version, correctly configured, and uses strong authentication.
+{{#is_federal}}As part of BOD 22-01, any “known exploited” findings, regardless of severity, need to be remediated within two weeks.{{/is_federal}}{{^is_federal}}CISA recommends remediating known exploited vulnerabilities, regardless of severity, within two weeks.{{/is_federal}}
```

Comment:
Replace smart quotes with regular quotes:
```suggestion
{{#is_federal}}As part of BOD 22-01, any "known exploited" findings, regardless of severity, need to be remediated within two weeks.{{/is_federal}}{{^is_federal}}CISA recommends remediating known exploited vulnerabilities, regardless of severity, within two weeks.{{/is_federal}}
```

Reply:
Hi Team, climber-girl CODEOWNERS file contains errors @ https://github.com/climber-girl/cyhy-mailer/blob/climber-girl-patch-1/.github/CODEOWNERS

Reply:
@dv4harr10 Yes because we do not have write permission for the forked repository. We can ignore this because we _do_ have write permission for _this_, the target repository.